applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_yield_surface.cpp

```diff
@@ -0,0 +1,71 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//
+
+#include "custom_constitutive/coulomb_yield_surface.h"
+#include "custom_constitutive/tension_cutoff.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities.h"
+#include "utilities/math_utils.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(TestCoulombYieldSurface, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    double           friction_angle = MathUtils<>::DegreesToRadians(45.0);
+    constexpr double cohesion       = 2.0;
+
+    CoulombYieldSurface coulombYieldSurface(friction_angle, cohesion, 0.0);
```

Comment:
According to the Kratos Style Guide:
```suggestion
    constexpr auto dilatancy_angle = MathUtils<>::DegreesToRadians(0.0);
    CoulombYieldSurface coulomb_yield_surface(friction_angle, cohesion, dilatancy_angle);
```

Also, I would give a name to the value of the final parameter (e.g. `dilatancy_angle`).